src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -222,12 +226,16 @@ def test_WINNF_FT_S_SIQ_3(self):
                                                           freq_range_high))
       channels_pal.append(filterChannelsByFrequencyRange(channels,
                                                          freq_range_pal))
-      channels_excluded.append(filterChannelsByFrequencyRange(channels,
-                                                              freq_range_excluded))
+      channels_excluded.append(
+          filterChannelsByFrequencyRange(channels, freq_range_excluded))
 
     # Checks for cbsd 1 (index 0)
     self.assertChannelsContainFrequencyRange(channels_low[0], freq_range_low)
-    self.assertChannelsOverlapFrequencyRange(channels_high[0], freq_range_high, constrain_low=True, constrain_high=False)
+    self.assertChannelsOverlapFrequencyRange(
+        channels_high[0],
+        freq_range_high_all,
+        constrain_low=True,
+        constrain_high=True)
```

Comment:
The original logic was to not constrain the high frequency edge of the channel to 3650 and thus will allow to go over it. The outer checking on all channels to not exceed 3700 will guarantee it is bounded. The modified logic will fail if the channel range is [3630 - F_high], when F_high < 3700, e.g. 3680, 3690 etc. Please double check. Can you give an example that the original logic will fail ?

Reply:
The logic was changed in today's WG4 meeting; please see issue #693 to understand the differences. Kambiz from Federated should also be aware of them if you want to double-check.

Reply:
I did not get this update. Do you mean that we changed the test spec to set the check to [F_high, 3700] ? If so, I have no concern with this fix.